plugins/woocommerce-blocks/assets/js/blocks-registry/product-collection/register-product-collection.tsx

```diff
@@ -375,6 +375,7 @@ export const __experimentalRegisterProductCollection = (
 		example: config.example,
 		scope: config.scope,
 		attributes: {
+			...config.attributes, // Allow users to pass extra attributes.
```

Comment:
In that case we can remove:

- `displayLayout: config.attributes?.displayLayout,` ([here](https://github.com/woocommerce/woocommerce/pull/53785/files#diff-c7bc52835382b93fa7671255cb9aae9e6eee453f93a53ed11f6e116646df7f5bR417))

- `queryContextIncludes: config.attributes?.queryContextIncludes,` ([here](https://github.com/woocommerce/woocommerce/pull/53785/files#diff-c7bc52835382b93fa7671255cb9aae9e6eee453f93a53ed11f6e116646df7f5bR419))

Reply:
Good point! I have removed the duplicate attribute assignments since we're now spreading all config.attributes at once. This makes the code cleaner and more maintainable. Thanks for pointing this out.